src/Resource.php

```diff
@@ -205,11 +205,11 @@ protected function mergeAttributes(array $attributes)
      *
      * @return Relationship[]
      */
-    public function getRelationships()
+    public function getRelationships($filter = true)
```

Comment:
without filtering `getRelationships` method is same as `buildRelationships` but has public accessibility.
If the method needs to be modified as `getFilteredRelationships` then calling places has to be updated.